wiki/Ranking_Criteria/osu!mania/de.md

```diff
@@ -1,91 +1,206 @@
+- **Langzeitige Slider-Velocity-Änderungen sollten zwischen 0,90x und 1,05x liegen.**
+
+Zusätzliche Richtlinien für *4 key Normal* Schwierigkeiten:
+
+- **Vermeide Chords mit mehr als 2 Noten.** Dies gilt auch für lange Noten-Patterns, die in mehr als zwei Spalten Noten enthalten. Es ist jedoch in Ordnung, ein Double zu verwenden, das mit der anderen Hand gespielt wird als die lange Note.
+
+Zusätzliche Richtlinien für *7 key Normal* Schwierigkeiten:
 
-- **osu!topus (AutoPlay) muss in allen Schwierigkeitsgraden die volle Punktzahl (1,000,000) erreichen können**.
-- **Es ist nicht erlaubt eine Column leer zu lassen**.
-- **Es ist nicht erlaubt mehr als 6 Tasten gleichzeitig auf einem Timing zu platzieren.**
+- **Vermeide Chords mit mehr als 3 Noten.** Dies gilt auch für lange Noten-Patterns, die in mehr als drei Spalten Noten enthalten. Es ist jedoch in Ordnung, ein Double zu verwenden, das mit der anderen Hand gespielt wird als die lange Note.
+- **Bracket-Patterns sollten sparsam verwendet werden.**
+- **Lange Noten, die mit benachbarten, gleichzeitig gehaltenen Tasten gespielt werden, sollten gleichzeitig losgelassen werden.** Lange Noten mit mehreren Release-Zeiten erfordern eine Koordination, die Spieler dieses Niveaus nicht vernünftig bewältigen können.
 
-Mit USB verbundene Tastaturen können mehr als 6 Tasten gleichzeitig nicht erkennen. Diese Regel wurde erstellt um den Spielspaß aller sicherzustellen *Das Ende eines Hold's ist davon nicht betroffen denn sie werden losgelassen, nicht gedrückt*
+### ![](/wiki/shared/diff/hard-m.png?20211215) Hard
 
-- **Es ist nicht erlaubt zwei Noten auf der gleichen Column auf dem selben Timing zu platzieren. Es muss immer ein Abstand zur nächsten Note auf der gleichen Column bestehen.**
-- **Letterboxing muss ausgeschaltet sein.** Letterboxing wird in osu!mania nicht unterstützt und gehört deshalb ausgeschaltet.
+#### Richtlinien
 
-### Difficulty Spread
+- **HP und OD Werte sollten nicht höher als 8 sein.**
+- **Taktteilung von 1/8 und höher sollten nicht aufeinanderfolgend verwendet werden.** Diese können für Spieler dieses Niveaus körperlich zu anstrengend sein. Eine Ausnahme von dieser Leitlinie wäre die Verwendung von 3 Grace-Noten, die sich auf höherem Niveau befinden (1/8, 1/12 und 1/16), da sie nur zur Akzentuierung von Gracing-Klängen verwendet werden.
+- **Lange Streams von 1/4-Noten sollten mindestens eine 1/2-Pause enthalten, wo dies natürlich nach jeder musikalischen Phrase möglich ist.** Diese können für Spieler dieses Niveaus körperlich zu anspruchsvoll sein.
+- **Chords innerhalb eines 1/4-Streams sollten mindestens ein 1/1-Beat voneinander entfernt sein.** Eine dichtere Verwendung von Chords ist für Spieler dieses Niveaus zu schwierig.
+- **Langzeitige Slider-Velocity-Änderungen sollten zwischen 0,8x und 1,10x liegen.**
```

Comment:
```suggestion
```
This line is already in 161 and this also corresponds with the English article. Comparing the line count of both languages (English and German) is a handy tool to superficially check if no content is forgotten (or doubled in this case).